main.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    return 'Hello, world2'
+
+def dataconversion():
+    base64img="""we get from online"""
```

Comment:
If the information between `"""` is a comment, the whole line should be commented to avoid compile errors

Also for remaining tasks, it's a good idea to use the key word `TODO` and specify who has to do it.

For example, if I had to do work, I might put `TODO (add36): Add json parsing to fetch image`